styles/main.css

```diff
@@ -199,6 +199,36 @@ h2.section-title {
   color: var(--ut-green);
 }
 
+
+/* BLOG SECTION */
+
+.article-head {
+  font-weight: 800;
+  font-size: 1.25rem;
+  font-family: var(--ut-font-sans);
+  color: #2BB673;
```

Comment:
please use variables for colors. Define the in the variables section, then use here